openwillis/measures/text/speech_attribute.py

```diff
@@ -168,34 +418,76 @@ def speech_characteristics(json_conf, language='en-us', speaker_label=None):
 
     Returns:
     ...........
-    tag_df: pandas dataframe
-        A dataframe containing speech tags
+    word_df: pandas dataframe
+        A dataframe containing word summary information
+    phrase_df: pandas dataframe
+        A dataframe containing phrase summary information
+    turn_df: pandas dataframe
+        A dataframe containing turn summary information
     summ_df: pandas dataframe
         A dataframe containing summary information on the speech
 
     ------------------------------------------------------------------------------------------------------
     """
-    measures = get_config()
-    tag_df, summ_df = create_empty_dataframes(measures)
+    measures = get_config(os.path.abspath(__file__), "text.json")
+    word_df, phrase_df, turn_df, summ_df = create_empty_dataframes(measures)
 
     try:
         if bool(json_conf):
             cutil.download_nltk_resources()
 
             if is_amazon_transcribe(json_conf):
-                text, filter_json = filter_transcribe(json_conf, speaker_label=speaker_label)
+                (
```

Comment:
as stated above, it should be returned as single list object

Reply:
https://github.com/bklynhlth/openwillis/pull/22/commits/21195f7b6bf9c238b8fc06322e1d896fc34501ed